src/libs/ReportUtils.js

```diff
@@ -1120,6 +1137,26 @@ function isIOUOwnedByCurrentUser(report, currentUserLogin, iouReports = {}) {
     return false;
 }
 
+function canSeeDefaultRoom(report, policies, betas) {
```

Comment:
We have excluded the `isArchivedRoom` case in this function, is that intentional?